Makefile

```diff
@@ -164,13 +164,16 @@ docker_build:
 # devstack-themed shortcuts
 dev.up: dev.up.redis
 	docker-compose up -d
+	docker exec -u 0 -it enterprise-subsidy.app pip-sync -q requirements/dev.txt requirements/private.* requirements/test.txt
```

Comment:
good idea!

Reply:
this could be a make command, which is chained on and thus re-used rather than redefined?